lms/djangoapps/certificates/views.py

```diff
@@ -121,3 +132,77 @@ def update_certificate(request):
         cert.save()
         return HttpResponse(json.dumps({'return_code': 0}),
                             mimetype='application/json')
+
+
+@login_required
+def render_html_view(request):
+    """
+    This view generates an HTML representation of the specified student's certificate
+    If a certificate is not available, we display a "Sorry!" screen instead
+    """
+    invalid_template_path = 'certificates/invalid.html'
+
+    # Feature Flag check
+    if not settings.FEATURES.get('CERTIFICATES_HTML_VIEW', False):
+        return render_to_response(invalid_template_path)
+
+    context = {}
+    course_id = request.GET.get('course', None)
```

Comment:
I'm actually not sure I want to use the course_id here -- the verify_uuid value is also available -- it really depends on how people are accessing this view from a UX perspective and what information  is available at that point.  For now I think we're simply looking to establish basic functionality in the LMS, however, so the course_id should suffice